perl/CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+Please read [CONTRIBUTING](https://github.com/cucumber/gherkin/blob/master/CONTRIBUTING.md) first.
+You should clone the [cucumber/gherkin](https://github.com/cucumber/gherkin) repo if you want
+to contribute.
+
+You will need `cpanm` installed on your system
```

Comment:
In that case, the CI server will need it too. Please update `/.travis.yml` (add an apt package I suppose)